api/app/signals/apps/api/v1/urls.py

```diff
@@ -63,6 +65,12 @@
         path('terms/categories/<str:slug>/sub_categories/<str:sub_slug>',
              ChildCategoryViewSet.as_view({'get': 'retrieve'}),
              name='category-detail'),
+        path('questions/categories/<str:slug>',
```

Comment:
To keep the URL in the same structure we already have I would suggest using: `terms/categories/<str:slug>/questions`

Reply:
See also my general comments about URL structure.

In short I would do something akin to: `/signals/v1/questions/?main_slug=<IETS>&sub_slug=<IETS ANDERS>` so that it is more future proof.